.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -6,7 +6,7 @@ labels: kind/bug
 assignees: ''
 
 ---
-
```

Comment:
Can we add this to all of our templates as well as our README?

Reply:
I was thinking that maybe we should add a security template that's JUST big scary words :)

Reply:
@dgerd I was thinking that anyone reporting something security-related would pick "bug" instead of "feature request" or "question," but maybe that's too optimistic. I'll add it to the other templates as well.

Reply:
I figure the more places it exists the greater the chance of someone seeing it and following the process.